bundle.yaml

```diff
@@ -390,10 +415,15 @@ spec:
                                             description: The key to select.
                                             type: string
                                           name:
+                                            default: ""
```

Comment:
I see lot of fields have defaults added with regenerate

Reply:
It might be related to https://github.com/kubernetes-sigs/controller-tools/issues/939?